### PR TITLE
Nukeops sheilded hardsuit has been slightly nerfed

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl: Supermichael777
tweak: The recharge rate and move speed of the syndicate shielded suit have been nerfed.
tweak: Functionality for delaying the time between suit charges has also been added and used.
tweak: Total change is a 40 second delay after hit with a 10 second pause between charges and it now moves as if in travel mode rather then combat mode.
/:cl:

### CODE REVIEW

**Comment by lzimann** [comment]
The mode speed is not really needed imo

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -479,6 +479,7 @@
 	var/recharge_delay = 200 //How long after we've been shot before we can start recharging. 20 seconds here
 	var/recharge_cooldown = 0 //Time since we've last been shot
 	var/recharge_rate = 1 //How quickly the shield recharges once it starts charging
+	var/recharge_speed = 5 //How long between charges after the sheid starts reacharging
```
Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/hardsuit.dm]: This var isnt needed at all

**Comment by KorPhaeron** [comment]
Why are you doing 2 different nerfs at once instead of trying one and then adding another later if its still needed?

**Comment by Lumipharon** [comment]
speed nerfs kill items.
No on uses items with speed nerfs unless that absolutely have to (which is why syndie hardsuits have fast mode...)

Shielded hardsuit is indeed unfair as fuck, but that's a terrible nerf.

**Comment by Steelpoint** [comment]
How much is the Shielded Hardsuit again? Wasn't it about 20 or 30 TC's?

Whatever the case is, reducing its move speed is functionally identical to removing it from the game.

**Comment by Dorsisdwarf** [comment]
Travel mode is faster than combat speed, isn't it?

**Comment by lzimann** [comment]
Nope, travel mode slows you down but has space protection, while combat removes this protection but has more speed(and I think it also has more armor, but not 100% on this)

**Comment by CPTANT** [comment]
fuck speed nerfs.

**Comment by Lumipharon** [comment]
Also on the topic of the hardsuits, can someone please just unfuck them?
Why do nukeop hardsuits have lights when they all have NIGHTVISION?
Why do they have lights only in the mode designed to be in SPACE, which is COMPLETELY ILLUMINATED ANYWAY?

It should just be a simple toggle between space worthiness and gofast.

**Comment by PKPenguin321** [comment]
hey here's a question why do you care if it has lights

**Comment by Lumipharon** [comment]
Uwot?

You have NVG's. The crew does not.
Lights help the crew find you, and give literally, LITERALLY no benefit what so ever to the nuke ops.

The most practical, and common situation, is nuke ops bomb comms.
Crew swarm comms area to fight ops in the dark, somewhat airless maint areas.
Nuke ops are glowing in the dark if they want to avoid cold/pressure damage, or hidden in the dark but taken damage from space exposure.

I brought this shit up when pap first added it, and it just got hand waved away.

**Comment by Supermichael777** [comment]
Basically before you were Sanic and could into space and basically required an auto shottie or literal a firing squad to take down as nothing else can hurt you enough quickly enough to take you out before you yakked into space. You effectively were a highly mobile mech-level threat at the cost of the joke of a machine gun. this makes ditching your team to go jungle some grey-shirts less viable as you are to slow to fight solo but durable enough to be point-man. this plus e-shield + no-slips = gg for everyone ever. the biggest liability you were facing was getting hit by your buddies spray and tazed and that was extremely unlikely. I wont touch the hard-suit headlights as that's outside this mild nerf.

**Comment by KorPhaeron** [comment]
This PR is overkill and the vars are done wrong anyway

**Pull request closed without merging**